flake.nix

```diff
@@ -33,6 +33,13 @@
       ```
     */
 
+    # We need this until NixOS/nixpkgs#327670 lands in nixos-unstable & we update the lockfile
+    nixpkgs-unstable.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+    nuschtosSearch = {
+      url = "github:NuschtOS/search";
+      inputs.nixpkgs.follows = "nixpkgs-unstable";
+    };
```

Comment:
How long would the wait? Is it worth delaying this PR?

Reply:
Not very long I think, a few days

Reply:
Unless it gets delayed, let's hold off so we don't have to add additional inputs and additional arguments to the docs package.

Reply:
Oh, I missed that the comment was referencing needing a newer nixpkgs lock _not_ needing the nuschtos flake input. I think that invalidates some of my other feedback.